langchain/chains/vector_db_qa/multi.py

```diff
@@ -0,0 +1,45 @@
+"""Chain for question-answering against multiple vector databases."""
+from __future__ import annotations
+
+from typing import Dict, List, Optional
+
+from langchain.chains.vector_db_qa.base import VectorDBQA
+from langchain.vectorstores.base import VectorStore
+
+
+class MultiVectorDBQA(VectorDBQA):
+    """Chain for question-answering against multiple vector database.
+
+    Example:
+        .. code-block:: python
+
+            from langchain import OpenAI, VectorDBQA
+            from langchain.faiss import FAISS
+            vectordb1 = FAISS(...)
+            vectordb2 = FAISS(...)
+            vectordbQA = VectorDBQA(llm=OpenAI(), vectorstore=[vectordb1, vectordb2])
+
+    """
+
+    # This is just to remove the singular field from base class
+    vectorstore: Optional[VectorStore]  # type: ignore
```

Comment:
hm i dont love this

two ideas:

1. we could make a base class and then have VectorDBQA and MultiVectorDBQA inheirit from that
2. we could make a VectorStore that is really a list of vector stores, and then do the similarity search inside there! that might actually result in less code changes, and is similar to how you did memory

thoughts?

Reply:
to be clear the part i dont love is having this argument thats not used/ignored

Reply:
yup, option 1 is the recommended way as well but I didn't want to make too big changes

Reply:
option 2 may be less changes - plus would actually be more generally useful! that way it could be used in all places where vectorstores are used (eg example selection, etc)

thoughts on doing option 2?

Reply:
yup, that sounds good :)